### ISSUE TITLE
`Image` should have a way to not generate an accessibility node

### ISSUE THREAD

**Issue by waywardmonkeys** [linebender/xilem#1136]
Some images are decorative and don't need to be present in the AccessKit node tree. There should be a way to prevent it from appearing.

I think this needs changes to the accessibility pass to provide a way to opt out of this process (so I haven't flagged it right now as a good first issue / help wanted).

**Comment by DataTriny** [comment]
@PoignardAzur as you pointed out, the `is_hidden` node property would be the preferred way to filter out an image node that is irrelevant to assistive technologies (does not carry information or isn't interactive). You want to avoid having an image node without a label as much as possible.

Even better if you can just not include these nodes in AccessKit tree updates, but otherwise the `is_hidden` property will do the job.
